### ISSUE TITLE
Run refparse JSON example in README is out of date

### ISSUE THREAD

**Issue by lizgzil** [wellcometrust/policytool#193]
In the README the example of running `refparse` includes using `"s3://datalabs-data/scraper-results/msf/20190117.json"`. This won't work anymore since the newer code relies on a `file_hash` column, but in this json the column is just called `hash`.

Furthermore, the old format didn't have the section texts as lists:
`"sections":{"Reference": '....', "Bibliograph": '....', "Endnote": '....'}`
Now:
`"sections":{"Reference": ['....'], "Bibliograph": ['....'], "Endnote": ['....']}`

Do we have a suitably new JSON that will work in S3?

**Temporary hack whilst we don't have a new format JSON**
- Change `file_hash` to `hash` in `refparse.py` and `/utils/file_manager.py`
- Old JSON version is not a list, so remove things which suggest this in `refparse.py`
namely, around line 57 comment out:
```
assert isinstance(section_list, list)
for section in section_list:
```

**Comment by lizgzil** [comment]
the data is in a `.json.gz` in each organisation folder here: s3://datalabs-dev/reach-airflow/output/policy/parsed-pdfs/

**Comment by nsorros** [comment]
Do we need to close this? Or we need to change the README first?

**Comment by lizgzil** [comment]
change the README to use the `datalabs-dev` location I think. I can work on this this week

**Comment by nsorros** [comment]
Go for it 👍

**Comment by lizgzil** [comment]
fixed in #245
